app/questionnaire/router.py

```diff
@@ -245,7 +266,7 @@ def _get_allowable_path(self, routing_path):
 
         return allowable_path
 
-    def get_enabled_section_keys(self):
+    def get_enabled_section_keys(self) -> Generator:
```

Comment:
We should specify the yield type. Should be tuple of two strings.

Reply:
The first yeld returns tuple of [Any, Any] the second tuple of [Any, None] so I left it without type of tuple contents, so `Generator[tuple, None, None]` for now..

Reply:
This wouldn't be `tuple[Any, str]` but `tuple[str, str]` once https://github.com/ONSdigital/eq-questionnaire-runner/pull/719#discussion_r753114992 is fixed.

Reply:
So it's not always `tuple[str, str]` as the Generator returns a tuple without `list_item_id` sometimes so I had to make it `Generator[Union[tuple[str, None], tuple[str, str]], None, None]` and in other places that call this method: `tuple[str, Optional[str]]`. Thanks!

Reply:
That's fine, it would be clearer to use `Optional` so `Generator[tuple[str, Optiona[str]], None, None]`

Reply:
Changed to `Generator[tuple[str, Optiona[str]], None, None]`, thanks!